packages/vanilla/src/index.ts

```diff
@@ -58,15 +64,7 @@ const loadScript = () => {
   return parallelPromise
 }
 
-// Wait a tick and then try to load the script.  This is done
-// to load the SDK upon library import.
-Promise.resolve()
-  .then(() => loadScript())
-  .catch(console.warn)
-
-let loadCalled = false
-
-export const loadParallel: LoadParallel = (config: ParallelConfig) => {
+export const loadParallel = (config: ParallelConfig): Promise<Parallel | null> => {
```

Comment:
The type specification for `loadParallel` isn't used anywhere else, so there's no real need to extract it as its own type.